api/tests/integration/application/schooling-registration-dependent-users/index_test.js

```diff
@@ -116,6 +116,63 @@ describe('Integration | Application | Route | schooling-registration-dependent-u
       // then
       expect(response.statusCode).to.equal(400);
     });
+
+    context('when username is not valid, it should return 400', () => {
+
+      it('if username is an email', async () => {
```

Comment:
**Suggestion : **Les noms de tests font plutôt penser à des noms de contexts.
Ils nous renseignent sur les conditions particulières dans lesquelles on se place pour tester (ici, le username ressemble à un email), mais ne nous explique pas quel comportement on attend.

Je propose deux versions :
- Soit écrire des sous-contextes avant chaque it 
```suggestion
     context('when username is an email', () => {
       
       it ('should return 400', async () => {
         ...
       }
     }
```

- Soit directement dans les it comme avant vu qu'il n'y aurait qu'un cas de test dans chaque sous-contexte
```suggestion
      it('should return 400 when username is an email', async () => {
```

Reply:
Je prend la deuxième version : 

context( 'when username is not valid' )
it( 'should return 400 when username is an email' )